### PR TITLE
refactor: capitalize Infrarouge, add abbreviations

### LINKED ISSUE #51: Capitalize Infrared and Ultraviolet, and add [IR], [UV]
+ translate Play (button tooltip message)

### PR DESCRIPTION
**Pull request by pyphilia**
close #51

### CODE REVIEW

File: src/components/lab/SpectrumBar.js [hunk 1]
```diff
@@ -143,6 +143,7 @@ const SpectrumBar = ({
           (SPECTRUM_BAR_HEIGHT - SPECTRUM_BAR_LABELS_FONT_SIZE) / 2
         }
         fill={ULTRAVIOLET_BAR_LABEL_COLOR}
+        align="right"
```
Comment (hagoptaminian) [inline src/components/lab/SpectrumBar.js:146]: I checked and `align="right"` doesn't seem to be affecting the position of this Text element. Would just remove...

**Review by hagoptaminian — COMMENTED** [review]
small comment but looks good

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Just echoing what Hagop mentioned in the comment.

File: src/components/lab/SpectrumBar.js (hunk 1 above)
Comment (pyphilia) [reply src/components/lab/SpectrumBar.js:146]: It's true that alone `align=true` does not do much alone. But together with `width` it allows for the text to be more "dynamic" and always aligned to the right (so adding more or less characters won't lead to adjust some factor constant). I did some small refactors around it so it works fully!

**Pull request merged**